Data-1.13/TableData/Items/Explosives.xml

```diff
@@ -1981,9 +1981,9 @@
 		<ubHorizontalDegree>0</ubHorizontalDegree>
 		<ubVerticalDegree>0</ubVerticalDegree>
 	</EXPLOSIVE>
-	<EXPLOSIVE><!-- roof collapse-->
+	<EXPLOSIVE><!-- roof collapse-->		<!-- Pr0ph3t: <ubType>12 is FIRE RETARDANT, Setting to <ubType>11 for DEBRIS SMOKE, Jan 15 2023 -->
```

Comment:
I'd discard this comment from the xml file. No need to clutter it with something like that when changing something.

Reply:
How do I do this

Reply:
I THINK it is good now ?